packages/sdk/src/conversation/interface.ts

```diff
@@ -163,6 +171,36 @@ export interface TeamsFxBotCommandHandler {
   ): Promise<string | Partial<Activity> | void>;
 }
 
+/**
+ * Interface for a command handler that can process sso command to a TeamsFx bot and return a response.
+ */
+export interface TeamsFxBotSsoCommandHandler {
```

Comment:
You can add yourself to sdk/conversation code owner  : )

Reply:
Sure